remoting/src/main/java/org/apache/rocketmq/remoting/protocol/RemotingCommand.java

```diff
@@ -81,6 +81,8 @@
 
     private transient byte[] body;
 
+    private Object payload;
```

Comment:
why do you want to add a new property here? and it would be nice if we can use byte[] body instead.